pfcpiface/conn.go

```diff
@@ -74,26 +77,29 @@ func (pConn *PFCPConn) startHeartBeatMonitor() {
 	hbCtx, hbCancel := context.WithCancel(pConn.ctx)
 	pConn.hbCtxCancel = hbCancel
 
-	log.Infoln("Starting Heartbeat timer")
+	hbLog := log.WithFields(log.Fields{
+		"interval": pConn.upf.hbInterval,
+	})
+	hbLog.Infoln("Starting Heartbeat timer")
 
-	heartBeatTimer := time.NewTimer(pConn.upf.hbInterval)
+	heartBeatExpiryTimer := time.NewTicker(pConn.upf.hbInterval)
 
 	for {
 		select {
 		case <-hbCtx.Done():
 			log.Infoln("Cancel HeartBeat Timer", pConn.RemoteAddr().String())
```

Comment:
Should all `log.` calls replaced with `hbLog`?
Can we just shadow `log` above?

Reply:
I guess the `interval` is only required when starting timer, but I modified a bit the style to make it more clean.

Reply:
I actually have no idea what the best practice around context loggers is, but I like the idea and think we should use them more often.